File: RoRServer/src/main/java/models/game/EditorPlayer.java
package models.game;

import java.util.UUID;

import communication.MessageInformation;

public class EditorPlayer extends Player{

	public EditorPlayer(String sessionName, String name, UUID id, boolean isHost) {
		super(sessionName, name, id, isHost);
		notifyCreated();
	}

	private void notifyCreated() {
    	MessageInformation messageInfo = new MessageInformation("CreatePlayer");
    	messageInfo.putValue("playerId", getId());
    	messageInfo.putValue("playerName", getName());
    	messageInfo.putValue("isHost", getIsHost());
    	notifyChange(messageInfo);
    }
}

File: RoRClient/Models/Game/EditorPlayer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RoRClient.Models.Game
{
    class EditorPlayer : Player
    {

        public EditorPlayer(Guid id, String name, bool isHost) : base(id, name, isHost)
        { 
        }
    }
}
